src/Raven.Server/Documents/Indexes/Workers/Cleanup/TombstoneIndexItem.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using Raven.Server.ServerWide.Context;
+using Raven.Server.Utils;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Documents.Indexes.Workers.Cleanup
+{
+    public class TombstoneIndexItem : IDisposable
+    {
+        public IndexItemType Type;
+        public LazyStringValue LowerId;
+        public LazyStringValue LuceneKey;
```

Comment:
Are we going to have a separate property for Corax? If not then why making it "Lucene specific"?

Reply:
Does it mean we'll have:

```
public LazyStringValue LuceneKey;
public LazyStringValue CoraxKey;
```

? If yes, then why we can't handle it via single field?

Reply:
I'm not sure if we're currently handling all the existing types for Corax. For example, the `TimeSeriesIndexItem` is still using `LuceneKey` in v6.x (unless I'm missing something). If that's the case, we might need to discuss the proper way to handle it and combine the fields :)

Reply:
> I'm not sure if we're currently handling all the existing types for Corax

Can you please verify that and open relevant tickets if needed?